mqml/tests/test_conductresist.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from mqml.parameter.conductresist import ConductResist
+from qcodes.instrument.base import Instrument
+import numpy as np
+
+@pytest.fixture(autouse=True)
+def close_all_instruments():
+    """Makes sure that after startup and teardown all instruments are closed"""
+    Instrument.close_all()
+    yield
+    Instrument.close_all()
+
+def test_get_static_parameters():
+    "This tests the initial values of static parameters in the class"
```

Comment:
What do you understand by static in this context?

Reply:
Should be changed to initial values.